main.cpp

```diff
@@ -66,23 +66,10 @@ int main(int argc, char *argv[])
 	capsViewer.show();
 
 	// Check for capability list xml
-	QFileInfo capsXmlFile("./capslist.xml");
-	if (!capsXmlFile.exists()) {
-		QMessageBox::StandardButton reply;
-		reply = QMessageBox::question(&capsViewer, "File missing", "glCapsViewer could not locate the xml file containing the OpenGL capability list!\n\nDo you want to download it?", QMessageBox::Yes | QMessageBox::No);

```

Comment:
Any reason for the removal of the file download here?

Reply:
Makes sense. The idea behind this was that I could upload a new file, but haven't done so in years. So if it's now part of the binary itself it's fine to remove the download stuff.